front-admin/README.md

```diff
@@ -1,46 +1,45 @@
-# Getting Started with Create React App
+# Bem vindo ao front-admin
+ 
+ Esse é o projeto front-end admin criado para os usuários admins que vão moderar a plataforma copo cheio. Nele é utilizado as tecnologias React TS, React-testing-library e Node para seu desenvolvimento. Além disso, utilizamos a metodologia do Atomic Design para estruturar nosso padrão de projeto e nossa estrutura de pasta.
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Testando
 
-## Available Scripts
+Para testar a aplicação rode o comando 
 
-In the project directory, you can run:
+```
+npm test
+```
 
-### `npm start`
+Caso queira rodar um arquivo especifico rode o comando:
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+```
+npm test __nome_do_arquivo__
+```
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## Rodando a aplicação
 
-### `npm test`
+Para rodar a aplicação é necessário instalar os pacotes de dependencia dentro da pasta front-admin, dessa forma rode o comando:
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+```
+$ cd front-admin
+```
 
-### `npm run build`
+Depois rode:
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+```
+$ npm install
+```
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+Em seguida rode o comando do docker:
```

Comment:
Se for usar o comando docker não precisa rodar npm install. O docker ja faz isso